forms-flow-web/src/resourceBundles/formio_resourceBundles.js

```diff
@@ -5156,8 +5157,7 @@ export const formio_resourceBundles = {
     "All Users": "Alle Nutzer",
     "Specific User Group": "Spezifische Benutzergruppe",
     "Applicable for Designer Roled Users only.": "Gilt nur für Benutzer mit Designer-Rolle.",
-    "Applicable for Client and Reviewer Roled Users only.": "Gilt nur für Benutzer mit Kunden- und Prüferrollen."
-
-
+    "Applicable for Client and Reviewer Roled Users only.": "Gilt nur für Benutzer mit Kunden- und Prüferrollen.",
```

Comment:
is this translation is related formio or our application? if it is based on us then please give move this to micro-front-end repo